CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `GetTheatre` API
 - `GetUnitType` API
 
+### Changed
+- Improved reliability of error handling.
```

Comment:
Nitpick. Could we expand on this where we mention that all errors are now logged only to `grpc.log`